docs/hooks/order-select.md

```diff
@@ -0,0 +1,366 @@
+# `order-select`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 0.1.0
+
+## Workflow
+
+The `order-select` hook fires when a clinician selects one or more orders to place for a patient. 
+This hook is among the first workflow events for an order entering a draft status. 
+The context of this hook may include defaulted order details 
+as it occurs immediately upon the clinician selecting the order from the order catalogue of the CPOE, 
+but before she manually specifies order details (e.g. dose, quanity, route, etc). 
+Additionally, the context may include previously selected orders that are not yet signed from the same ordering session. 
+The `order-select` hook occurs after the clinician selects the order, but before she fills out (configures) the order. 
+
+## Context
+
+The context of this hook distinguishes between the list of unsigned orders from the clinician's ordering session, and one or orders just added to this list. The `selections` field contains a list of ids of these newly selected orders; the `orders` Bundle contains an entry for all unsigned orders from this session, including newly selected orders.
+
+Field | Optionality | Prefetch Token | Type | Description
+----- | -------- | ---- | ---- | ----
+`userId` | REQUIRED | Yes | *string* | The id of the current user.<br />For this hook, the user is expected to be of type [Practitioner](https://www.hl7.org/fhir/practitioner.html).<br />For example, `Practitioner/123`
+`patientId` | REQUIRED | Yes | *string* |  The FHIR `Patient.id` of the current patient in context
+`encounterId` | OPTIONAL | Yes | *string* |  The FHIR `Encounter.id` of the current encounter in context
+`selections` | REQUIRED | No| *array* | The FHIR id of the newly selected order(s).<br />The `selections` field references FHIR resources in the `orders` Bundle. For example, `MedicationRequest/103`.
+`orders` | REQUIRED | No | *object* | DSTU2 - FHIR Bundle of MedicationOrder, DiagnosticOrder, DeviceUseRequest, ReferralRequest, ProcedureRequest, NutritionOrder, VisionPrescription with _draft_ status <br/> STU3 - FHIR Bundle of MedicationRequest, ReferralRequest, ProcedureRequest, NutritionOrder, VisionPrescription with _draft_ status
```

Comment:
Should this be renamed as `draftOrders`?

Reply:
`order-review` calls this `orders`, does that change your mind?

Reply:
We talked through this on the CDS call, there was general support for the name `draftOrders` over the current `orders`. This has been updated in `order-sign` and updating now in `order-select` PRs.